pyproject.toml

```diff
@@ -4,6 +4,7 @@ requires = [
     "wheel>=0.30",
     "jupyter_packaging~=0.9.0",
     "jupyterlab~=3.0",
+    "jupyter_client~=6.2.0"
```

Comment:
This shouldn't be needed if https://github.com/bloomberg/ipydatagrid/pull/274 gets in